packages/participants-svc/src/application/routes.ts

```diff
@@ -110,6 +111,7 @@ export class ExpressRoutes {
         // account
         this._mainRouter.get("/participants/:id/accounts", this._accountsByParticipantId.bind(this));
         this._mainRouter.post("/participants/:id/account", this._participantAccountCreate.bind(this));
+        this._mainRouter.post("/participants/:id/account/:accountChangeRequestId/approve", this._participantAccountApprove.bind(this));
```

Comment:
Aren't we missing a POST to "/participants/:id/accountchangerequests" to send the bank ID and/or name change request? The one that creates the change request.

Then the approval should be a POST to "/participants/:id/accountchangerequests/:changereqid/approve"

Same as the net debit cap management below

Reply:
I think, we don't need to pass bankId or name in the endpoint. We just passed changeRequestId. Bank info will include in the body.
For the approve, I updated as per your suggestion.